Sources/SwiftInspectorVisitors/FileVisitor.swift

```diff
@@ -55,39 +55,15 @@ public final class FileVisitor: SyntaxVisitor {
   }
 
   public override func visit(_ node: StructDeclSyntax) -> SyntaxVisitorContinueKind {
-    let structVisitor = StructVisitor()
```

Comment:
These three method bodies looked _exactly_ the same. And now that we have a single protocol representing a `NestableDeclSyntax`, we could write this code once and reuse it.